db/migrate/20240318171344_migrate_councils_to_user_groups.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class MigrateCouncilsToUserGroups < ActiveRecord::Migration[7.1]
+  def change
```

Comment:
The name of the migration implies you're migrating all councils. In reality, you're just migrating one specific council by a hard-coded ID.

This may "accidentally" work because we know as of 2024-03-25 there is only one council, but shouldn't there be some `UserGroup.all_councils` constant or similar, that you can loop over?

Reply:
There was `Team.all_councils`, but that was removed in this PR because it's no longer used. Also, even inside `all_councils`, the list was hard-coded. So I thought not using that will be the easiest way.

Also, even if we bring back `all_councils` in this PR, we might remove it after few PRs. And if somebody is running pending migration, there might occur a case where `all_councils` method doesn't exist.

Reply:
Hard-coding the list is okay if there's a good reason for it. And formal definitions in our Motions (which are outside of the realm of code) are a good reason for hard-coding stuff. Like "We have a Motion that defines these 3 councils currently exist". That's a bureaucratic definition that you just have to hard-code.

Your argument about running migrations much later also applies to your current approach of `Team.c_find_by_id!`. What if that method doesn't exist anymore in the feature because two months from now, your project is done and the `Team` entity won't exist in our code anymore?

Reply:
(My point is: All migrations have inherent weaknesses.)

Reply:
Okay makes sense. I've changed it to a list with just WAC in the array. Hope this is fine now, let me know if I need to re-introduce Team.all_councils method.